Mathlib/Tactic/Finiteness.lean

```diff
@@ -0,0 +1,64 @@
+/-
+Copyright (c) 2023 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
+import Mathlib.Tactic.Positivity.Core
+
+/-!
+# Finiteness tactic
+
+This file implements a basic `finiteness` tactic, designed to solve goals of the form `*** < ∞` and
+(equivalently) `*** ≠ ∞` in the extended nonnegative reals (`ENNReal`, aka `ℝ≥0∞`).
+
+It works recursively according to the syntax of the expression. It is implemented as an `aesop` rule
+set.
+
+## Syntax
+
+Standard `aesop` syntax apply. Namely one can write
+* `finiteness (add simp [lemma1, lemma2])` to make the `simp` call in `finiteness` use `lemma1`,
```

Comment:
Doesn't the `enableSimp := false` disable this?

Reply:
Right, well spotted